lib/model/compose.dart

```diff
@@ -101,18 +126,33 @@ String wrapWithBacktickFence({required String content, String? infoString}) {
   return resultBuffer.toString();
 }
 
-/// An @-mention, like @**Chris Bobbe|13313**.
+/// An @user-mention, like @**Chris Bobbe|13313**.
```

Comment:
nit: "A @user-mention"

Reply:
I'd actually pronounce this as "an at-user mention" so would use "an".

(But I think it could be clearer with a few more words: "An @-mention of an individual user", similar to the doc on UserMentionAutocompleteResult.)